src/main/java/api/clients/ApiBoardClient.java

```diff
@@ -35,4 +37,14 @@ public ValidatableResponse deleteExistingBoard(String boardId, int expectedStatu
                 .then()
                 .statusCode(expectedStatusCode);
     }
+
+    public ValidatableResponse updateBoard(String key, String value, String boardId, int expectedStatusCode) {
+        return given()
+                .spec(requestSpec)
+                .pathParam("id", boardId)
+                .body(Map.of(key, value))
+                .put("/1/boards/{id}")
+                .then()
+                .statusCode(expectedStatusCode);
+    }
 }
```

Comment:
You use your ID twice(in `pathParam` and `put`
Also, your mehtod can accept only one key and value. Create a separate builder for the update and submit it here